commands/commands.go

```diff
@@ -55,8 +60,13 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 	log.Debugf("%s.Run start", c.Name)
 
 	cmd := ArgsToCmd(c.Exec, c.Args)
```

Comment:
The function signature is `ArgsToCmd(executable string, args []string) *exec.Cmd`, so it takes command arguments and makes a `Cmd` out of them. The name is probably lispy though (I'm guilty of that sometimes)... maybe more idiomatic would be `NewCmd`? Or maybe just inline it, as I think there's only one or two callers at this point.